apps/condo/migrations/20220705094338-0143_remove_ticket_categoryclassifier_and_more.js

```diff
@@ -0,0 +1,168 @@
+
+exports.up = async (knex) => {
+    await knex.raw(`
+    BEGIN;
+--
+-- Remove field categoryClassifier from ticket
+--
+SET CONSTRAINTS "Ticket_categoryClassifier_e01aafa7_fk_TicketCat" IMMEDIATE; ALTER TABLE "Ticket" DROP CONSTRAINT "Ticket_categoryClassifier_e01aafa7_fk_TicketCat";
+ALTER TABLE "Ticket" DROP COLUMN "categoryClassifier" CASCADE;
+--
+-- Remove field placeClassifier from ticket
+--
+SET CONSTRAINTS "Ticket_placeClassifier_01e3ec3a_fk_TicketPlaceClassifier_id" IMMEDIATE; ALTER TABLE "Ticket" DROP CONSTRAINT "Ticket_placeClassifier_01e3ec3a_fk_TicketPlaceClassifier_id";
+ALTER TABLE "Ticket" DROP COLUMN "placeClassifier" CASCADE;
```

Comment:
With this migration you will just wipe out all information on classifier. Data transfer SQL-queries should be executed first and then you can drop all unnecessary columns.